tests/conformance-tests/src/main.rs

```diff
@@ -0,0 +1,40 @@
+use testing_framework::runtimes::spin_cli::SpinCli;
+
+fn main() {
+    let spin_binary: std::path::PathBuf = std::env::args()
+        .nth(1)
+        .expect("expected first argument to be path to spin binary")
+        .into();
+    let tests_dir = conformance_tests::download_tests().unwrap();
+
+    for test in conformance_tests::tests(&tests_dir).unwrap() {
+        let env_config = SpinCli::config(
+            spin_binary.clone(),
+            [],
+            move |e| {
+                e.copy_into(&test.manifest, "spin.toml")?;
+                e.copy_into(&test.component, test.component.file_name().unwrap())?;
+                Ok(())
+            },
+            test_environment::services::ServicesConfig::none(),
+            testing_framework::runtimes::SpinAppType::Http,
+        );
```

Comment:
Maybe its just because I'm looking at this without the help of a IDE, but its a bit hard to understand what some of these args mean (`[]`, the closure). Could/should this be a struct?